tests.py

```diff
@@ -188,3 +188,15 @@ def test_sip_api_already_set():
             import sip
             sip.setapi("QString", 1)
             assert_raises(ImportError, __import__, "Qt")
+
+
+def test_qtwebkit_pyside():
```

Comment:
These tests aren't testing anything of value but rather adds bulk to the project. If we want to test that members of Qt.py are mapped (or not) we should probably do so all together in a single test function.